### ISSUE TITLE
Being able to mute notification for a conversation

### ISSUE THREAD

**Issue by IpsmLorem** [PeerioTechnologies/peerio-client#86]
It would be nice to be able to mute notification for a conversation

**Comment by skylarnagao** [comment]
Duplicate #40

**Comment by IpsmLorem** [comment]
No. This is not a sound mute. 
I was talking about email notification mute on user chosen conversation.

For exemple, you have a group conversation with a lot of activity but you are merely interested by the topic and you must have a way to not be notified for new messages to reduce the "noise".

**Comment by skylarnagao** [comment]
To clarify, you mean the green-bar for an "unread message"?

**Comment by IpsmLorem** [comment]
No. i mean to not receive email notification for a user selected Messages in Peerio.
